docs/partial_source/deep_dive/16_ivy_frontends.rst

```diff
@@ -520,18 +522,17 @@ instance, and then passes the key and value as inputs to the corresponding funct
 correctly.
 
 
-As an example, we show how :class:`NativeClass` is used in the frontend test for the
-:func:`sum` function in the NumPy frontend:
+As an example, we show how :code:`NativeClass` is used in the frontend test for the
+:func:`sum` function in the NumPy frontend: 
 
 .. code-block:: python
-    # sum
-    Novalue = NativeClass(numpy._NoValue)
+
     @handle_cmd_line_args
     @given(
-        dtype_x_axis=_dtype_x_axis(available_dtypes=helpers.get_dtypes("float")),
+        dtype_x_axis=helpers.dtype_values_axis(available_dtypes=helpers.get_dtypes("float")),
         dtype=helpers.get_dtypes("float", full=False, none=True),
-        keep_dims= st.one_of (st.booleans(), Novalue),
-        initial=st.one_of(st.floats(), Novalue),
+        keep_dims=st.booleans(),
+        initial=st.one_of(st.floats(), st.none()),
```

Comment:
why have you removed the `Novalue` stuff? This was one of the points mentioned in the sentence above?

Reply:
I just used the recent version of it here (I wasn't sure about what happened to it) : https://github.com/unifyai/ivy/blob/8554cd304f9a35d39693e03878aab62a97acbcef/ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py#L18

Reply:
I've temporary reverted the changes for it.